client-tez/src/test/java/org/apache/tez/runtime/library/common/sort/buffer/WriteBufferManagerTest.java

```diff
@@ -1,388 +0,0 @@
-/*
- * Licensed to the Apache Software Foundation (ASF) under one or more
- * contributor license agreements.  See the NOTICE file distributed with
- * this work for additional information regarding copyright ownership.
- * The ASF licenses this file to You under the Apache License, Version 2.0
- * (the "License"); you may not use this file except in compliance with
- * the License.  You may obtain a copy of the License at
- *
- *    http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-package org.apache.tez.runtime.library.common.sort.buffer;
```

Comment:
Why do we need to delete so many tests?

Reply:
The test reserved now is enough.

Reply:
Why do you think it's enough?

Reply:
Merging [#948](https://app.codecov.io/gh/apache/incubator-uniffle/pull/948?src=pr&el=desc&utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=apache) (https://github.com/apache/incubator-uniffle/commit/fe0b7b15e881faf767727953e4e5654da811235c) into [master](https://app.codecov.io/gh/apache/incubator-uniffle/commit/fb394e5ee8b40c8ee176eaf8fc2e000e072a247f?el=desc&utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=apache) (https://github.com/apache/incubator-uniffle/commit/fb394e5ee8b40c8ee176eaf8fc2e000e072a247f) will increase coverage by 1.52%.

Reply:
> Merging [#948](https://app.codecov.io/gh/apache/incubator-uniffle/pull/948?src=pr&el=desc&utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=apache) ([fe0b7b1](https://github.com/apache/incubator-uniffle/commit/fe0b7b15e881faf767727953e4e5654da811235c)) into [master](https://app.codecov.io/gh/apache/incubator-uniffle/commit/fb394e5ee8b40c8ee176eaf8fc2e000e072a247f?el=desc&utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=apache) ([fb394e5](https://github.com/apache/incubator-uniffle/commit/fb394e5ee8b40c8ee176eaf8fc2e000e072a247f)) will increase coverage by 1.52%.

k8s operator tests are ignored on PR, the coverage message might be misleading.

Reply:
I can't accept to delete the UT if we don't delete some unused code. Coverage isn't accurate.

Reply:
It may be mistake to delete so many test file, and it reverted.